app/src/main/java/org/mozilla/focus/widget/DpadKeyMovementMethod.kt

```diff
@@ -0,0 +1,17 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
```

Comment:
Nit: We can delete this top line, it's an outdated convention that we don't need to track anymore.